Intersect (Core)/Extensions/VariableModExtensions.cs

```diff
@@ -0,0 +1,36 @@
+using System.Reflection;
+using Intersect.Attributes;
+using Intersect.Enums;
+
+namespace Intersect.Extensions;
+public static class VariableModExtensions
+{
+    public static VariableType GetRelatedVariableType(this VariableMod value)
+    {
+        if (!Enum.IsDefined(typeof(VariableMod), value))
```

Comment:
You don't need the `typeof()` parameter here, there's a generic single-parameter `IsDefined`